BUILDING.md

```diff
@@ -163,7 +163,8 @@ $ [sudo] make install
 
 Prerequisites:
 
-* [Python 2.6 or 2.7](https://www.python.org/downloads/)
+* [Python 2.6 or 2.7](https://www.python.org/downloads/)  
+  *Note*: It is recommanded to include the "Add python.exe to path" option
```

Comment:
I'm fine with https://github.com/nodejs/node/issues/13882 landing rather than this note.

I like spoon feeding devs when it comes to tooling (and not code) - I don't want the fact building is hard or unclear to prevent contributors from joining.